roomControl.cs

```diff
@@ -20,122 +21,143 @@ public roomControl()
             InitializeComponent();
         }
 
-        SqlConnection connectn = new SqlConnection("Data Source=LAPTOP-VBIOM4D2;Initial Catalog=Octopus;Integrated Security=True");
-        //Data Source=LAPTOP-VBIOM4D2;Initial Catalog=Octopus;Integrated Security=True
-       
+        private string ConnectionString = ConfigurationManager.ConnectionStrings["hotel.Properties.Settings.Setting"].ConnectionString;
+      
         private int convertID ;
-       
-       
-        private string price;
-        
-        private void aadd(int number)
+        private TimeSpan sumDay;
+        private void addingRoom(int number)
         {
-          
-           
-            
             convertID = Convert.ToInt32(txtResearchRoom.Text);
-            connectn.Open();
-            SqlCommand commandd = new SqlCommand(" select  price from customer where  ID = (" + convertID + ")", connectn);
-            commandd.ExecuteNonQuery();
-            SqlDataReader read = commandd.ExecuteReader();
-           
-            while (read.Read())
+            using (var connection = new SqlConnection(ConnectionString))
             {
                
-                price = read["price"].ToString();
-           }
-           
-            read.Close();
-            connectn.Close();
-            int howmanyday = 0;
-             int numb= 0;
-           
-            numb = Convert.ToInt32(price);
-            howmanyday = (numb/50);
-           connectn.Open(); 
-           SqlCommand insert = new SqlCommand("Insert into Room(ID, roomNo, roomColor, howManyDay, checkIn) values ('" + convertID + "','" + number + "', 'Salmon','" + howmanyday + "', GETDATE() )", connectn);
-           insert.ExecuteNonQuery();
-           connectn.Close();
+                connection.Open();
+                DateTime loginDate;
+                DateTime exitDate;
+
+                string timeQuery = "select  loginDate, exitDate from customer where  ID = @ID ";
+                SqlCommand sqlCommand = new SqlCommand(timeQuery, connection);
+                sqlCommand.Parameters.Add(new System.Data.SqlClient.SqlParameter("@ID", SqlDbType.Int, 3) { Value = convertID });
+                sqlCommand.ExecuteNonQuery();
```

Comment:
Buradaki `ExecuteNonQuery` neden var?

Reply:
aa evet update, insert, delete  gibi bir işlem yapmadığım için kaldırabilirim bunu. düzeltiyorum hemen :)